picard/album.py

```diff
@@ -33,6 +32,7 @@
 from picard.script import ScriptParser
 from picard.ui.item import Item
 from picard.util import format_time, mbid_validate
+from picard.util.imagelist import ImageList
```

Comment:
You forgot two occurences:
picard/ui/coverartbox.py:27:from picard.coverart.imagelist import ImageList
picard/ui/infodialog.py:29:from picard.coverart.imagelist import ImageList